website/docs/docs/stubs.md

```diff
@@ -52,13 +53,32 @@ For example, assuming you wanted to use the latest version of a particular contr
 https://<yourdomain>.pactflow.io/pacts/provider/:provider/consumer/:consumer/latest/stub
 ```
 
+## Configuration
+
+You can configure the behaviour of the stub service at runtime, using HTTP headers sent with the stub requests.
+
+| Header | Type | Description | Default |
+|--------|------|-------------|---------|
+| `pactflow-stub-cors` | boolean  | Automatically respond to OPTIONS requests and return default CORS headers | `true` |
+| `pactflow-stub-cors-referer` | boolean | Set the CORS Access-Control-Allow-Origin header to the Referer | true |
```

Comment:
Sets the CORS origin value to the hostname of the referer URL if set to true. If set to to false, or if there is no referer header, sets it to '*". Default is false.

Reply:
Updated, thx.